nixos/modules/services/networking/radicale.nix

```diff
@@ -84,9 +161,43 @@ in
         ));
         User = "radicale";
         Group = "radicale";
+        StateDirectory = "radicale/collections";
+        # Hardening
+        CapabilityBoundingSet = "";
```

Comment:
This needs `CAP_NET_BIND_SERVICE` if you want to use a port < 1024. How should we handle that?

Reply:
Best case: Socket activation, if supported.
Second best: Only pass it in if `port < 1024`.

I don't think python actively requests capabilities, so you'd need to put it into `AmbientCapabilities` as well.

Reply:
> Best case: Socket activation, if supported.
not yet: https://github.com/Kozea/Radicale/pull/1141

> Second best: Only pass it in if `port < 1024`.
Then we need a new service option because we shouldn't parse `settings.server.hosts`.
We can make it dependent on `bindLocalhost`.

Reply:
Actually, the current implementation doesn't have the permissions to bind to ports < 1024, so they should also be given by hand in the future.